collect_app/src/main/java/org/odk/collect/android/tasks/DownloadFormListTask.java

```diff
@@ -272,8 +272,8 @@ private boolean isXformsListNamespacedElement(Element e) {
                     }
                     if (downloadUrl == null || formName == null) {
                         String error =
-                                "Forms list entry " + Integer.toString(i)
-                                        + " is missing form name or url attribute";
+                                "Forms List Entry " + Integer.toString(i)
+                                        + " has missing or empty tag(s) : form or url";
```

Comment:
No space before colon. `url` is an attribute, not a tag. So something like "has missing or empty form name or url attribute" would be better